### DISCUSSION TITLE
Did you use any plugins that don't have any updated version that runs in 10.6.6?

### DISCUSSION THREAD

**Discussion by CoreMeltFx** [CommandPost/FCPCafe#121] [General Discussions]
If so what are they?  CoreMelt might be interesting in making replacements

**Comment by d-mac-648** [comment]
Haven't "updated" to 10.6.6 due to missing audio plugins, and other potential issues. So, it would be nice to have access to anything Apple foolishly removed if they don't restore them. Wondering if Apple did this to make it easier to release iPadOS FCP (and if they'll restore the missing functionality, similar to what they did with Pages, etc., when iOS versions first appeared).

Also still running Monterey 12.6.6 (on Intel Mac) to further avoid Apple Silicon issues with Ventura...

**Comment by hexeric** [comment]
no, all smooth and even smoother than before
